### PR TITLE
Autoscale scale test case improvement.

### PR DESCRIPTION
**Pull request by tiffanyn108**
Autoscale test case improvement for scaling up more than 2 endpoints.

Signed-off-by: tiffanyn108 <tiffanyn108@github.com>

### CODE REVIEW

File: ocs_ci/ocs/scale_noobaa_lib.py [hunk 1]
```diff
@@ -112,3 +114,44 @@ def cleanup(namespace):
...
+    """
+    obj = ocp.OCP()
+    hpa_resources = obj.exec_oc_cmd(
+        command=f"get hpa -n {namespace}", out_yaml_format=False
+    ).split(",")
+    for value in hpa_resources:
+        value = re.findall(r"(\d{1,3})%/(\d{1,3})%", value.strip())
+        value_list = [item for elem in value for item in elem]
+        hpa_cpu_utilization = int(value_list[0])
+        log.info(f"HPA CPU utilization by noobaa-endpoint is {hpa_cpu_utilization}%")
```
Comment (keesturam) [inline ocs_ci/ocs/scale_noobaa_lib.py:156]: IMO, the function should simply return the value and the logging needs to be done outside the function. This improves readability.

File: tests/e2e/scale/noobaa/test_scale_endpoint.py [hunk 2]
```diff
@@ -71,7 +82,10 @@ def _assert_endpoint_count(self, desired_count):
         )
 
     def test_scale_endpoint_and_respin_ceph_pods(
-        self, mcg_job_factory, resource_to_delete
+        self,
```
Comment (keesturam) [inline tests/e2e/scale/noobaa/test_scale_endpoint.py:85]: can we remove the `MAX_ENDPOINT_COUNT = 2` as use it as a parameter instead?

**Review by Avilir — DISMISSED** [review]
LGTM

File: ocs_ci/ocs/scale_noobaa_lib.py (hunk 1 above)
Comment (tiffanyn108) [reply ocs_ci/ocs/scale_noobaa_lib.py:156]: This can be changed. Fixed.

File: tests/e2e/scale/noobaa/test_scale_endpoint.py (hunk 2 above)
Comment (tiffanyn108) [reply tests/e2e/scale/noobaa/test_scale_endpoint.py:85]: This parameter is using to determine max replicas HPA deployment.  It can't be removed.

**Review by Avilir — APPROVED** [review]
LGTM

**Review by ramkiperiy — APPROVED** [review]

**Pull request merged**